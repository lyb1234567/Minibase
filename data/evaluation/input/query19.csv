'smith', 16
'anka', 8
'rhcp', 24
'bowie', 40
'pistols', 80
