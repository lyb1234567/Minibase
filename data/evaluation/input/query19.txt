Q(y,SUM(2*1*x*z)) :- S(x,y,z), 'mlp' != 'adbs'