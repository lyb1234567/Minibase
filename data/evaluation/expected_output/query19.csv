'smith', 16
'rhcp', 24
'pistols', 80
'bowie', 40
'anka', 8